lispy.el

```diff
@@ -8579,23 +8584,18 @@ When ARG is non-nil, unquote the current string."
     (define-key map (kbd "M-?") 'lispy-convolute-sexp)
     (define-key map (kbd "M-S") 'lispy-split)
     (define-key map (kbd "M-J") 'lispy-join)
-    (define-key map (kbd "{") 'lispy-braces)
-    (define-key map (kbd "}") 'lispy-brackets)
-    (define-key map (kbd "]") 'lispy-forward)
-    (define-key map (kbd "[") 'lispy-backward)
```

Comment:
These bindings directly conflict with bindings right above. The bindings defined first follow the same rules as paredit. The bindings defined here behave like lispy. I have followed paredit conventions, because cleverparens by default doesn't have special behaviour for square brackets.

I can make a separate evil-cp-lispy or evil-cp-paredit theme if desired or even remove all bindings except the ones with meta, so this can be combined with the themes for paredit, lispy or even parinfer depending on how someone wants [ and ] to behave. On the other hand, evil-cp/paredit users may want M-] and evil-cp/lispy users may want M-} to wrap something in square brackets ("M-]" is the standard binding in evil-cleverparens).